<?xml version="1.0"?>
<launch>
  <arg name="camera_file" default="$(find rct_examples)/config/kinect_camera_intr.yaml"/>
  <arg name="target_file" default="$(find rct_examples)/config/target_10x10.yaml"/>
  <arg name="image_path" default="$(find rct_examples)/data/test_set_10x10/images/0.png"/>

  <node pkg="rct_examples" type="solve_pnp_ex" name="solve_pnp_ex" output="screen">
    <rosparam command="load" file="$(arg camera_file)"/>
    <rosparam command="load" file="$(arg target_file)"/>
    <param name="image_path" value="$(arg image_path)"/>
  </node>
</launch>
